flybywire-aircraft-a320-neo/effects/LIGHT_A32NX_WingLargeVol.fx

```diff
@@ -65,11 +65,11 @@ MinProjSize=0.2
 
 [LightAttributes.0]
 Type=spot
-Size=0.1
-Range=20
-Intensity=50
+Size=0.8
+Range=100
+Intensity=100
 Softness=0.0
 SpotInner=0
-SpotOuter=22
-Volumetric=1
+SpotOuter=25
+Volumetric=0
```

Comment:
Why are we removing the volumetric effect on these lights? If that is inteded, all the properties of this light could be moved into the regular light. Se previous comment about positioning/adding more lights for sculpting this effect.